### PR TITLE
[WebRTC][TCSACR-430] Add WebRTC dotnet guide

### PR DESCRIPTION
**Pull request by hsgwon**
### Change Description ###
Add WebRTC dotnet guide

### API Changes ###
 - ACR: TCSACR-430/479/480/488/491/501/502/506/510

### CODE REVIEW

**Comment by hsgwon** [comment]
This PR is newly raised for https://github.com/Samsung/tizen-docs/pull/1747 and I update document as review's comment.

File: docs/application/dotnet/guides/multimedia/webrtc.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+    ...
+    await webRtc.StartAsync();
+    ```
+
+<a name="statistics_info"></a>
+## Get statistics information
+
+You can get the statistics information of the WebRTC.
+
+1. To get statistics information, use the `WebRTC.GetStatistics()`. This method is possible in the `WebRTCState.Playing` state.:
```
Comment (maliha-srbd) [inline docs/application/dotnet/guides/multimedia/webrtc.md:492]:
```suggestion
1. To get statistics information, use the `WebRTC.GetStatistics()`. This method is possible in the `WebRTCState.Playing` state:
```

File: docs/application/dotnet/guides/multimedia/webrtc.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+    ```
+    > [!NOTE]
+    > If the device ID is not valid, `StartAsync()` or `Start` will throw `InvalidOperationException` exception.
+
+<a name="data_channel"></a>
+## Control data channels
+
+You can create a data channel to the WebRTC. It is also possible to be notified when you have a new data channel requested by a remote peer. You can send, or receive data, to or from these data channels by using the APIs below.
+
+1. To create a data channel, use `WebRTCDataChannel` before calling `Start()` or `StartAsync()`:
```
Comment (maliha-srbd) [inline docs/application/dotnet/guides/multimedia/webrtc.md:187]: Hello @hsgwon , I have a question in Line# 187. Is the event `WebRTCDataChannel` in the correct form? Should I be `WebRTC.DataChannel` instead?
I hope you address my confusion here, thank you.

**Review by maliha-srbd — COMMENTED** [review]
Hello @hsgwon , hope you're doing well. I noticed that the **PR# 1747** was closed and that it has been reopened as **PR# 1750**. Thank you for incorporating all the suggested changes to it. 
I have reviewed this PR, and had a few more suggestions and concerns to add. Please check them at your convenience. Thank you.

File: docs/application/dotnet/guides/multimedia/webrtc.md (hunk 2 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/webrtc.md:187]: `WebRTCDataChannel` is not a event, it's class. 
Do I need to add a word `class` after `WebRTCDataChannel`?

File: docs/application/dotnet/guides/multimedia/webrtc.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+## Get statistics information
+
+You can get the statistics information of the WebRTC.
+
+1. To get statistics information, use the `WebRTC.GetStatistics()`. This method is possible in the `WebRTCState.Playing` state.:
+    ```csharp
+    ...
+    // WebRTCState is changed to Playing
+    ...
+    var stats = webRtc.GetStatistics(WebRTCStatisticsCategory.All);
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/webrtc.md:497]: I'll change `WebRTC.GetStatistics()` to `GetStatistics()`. Is it OK?
(Actually it's not static method but, I intended `GetStatistics()` method of `WebRTC` class.)

File: docs/application/dotnet/guides/multimedia/webrtc.md (hunk 2 above)
Comment (maliha-srbd) [reply docs/application/dotnet/guides/multimedia/webrtc.md:187]: Thank you for the clarification. Yes, I think adding the word **class** after **WebRTCDataChannel** would make it clearer, so I'd suggest you do that. Thank you.

File: docs/application/dotnet/guides/multimedia/webrtc.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (maliha-srbd) [reply docs/application/dotnet/guides/multimedia/webrtc.md:497]: Yes, I think changing it to **GetStatistics()** would make it clearer. Thank you.

File: docs/application/dotnet/guides/multimedia/webrtc.md (hunk 1 above)
Comment (maliha-srbd) [reply docs/application/dotnet/guides/multimedia/webrtc.md:492]: The change suggested in Line# 492 does not appear in the staging site. Please check again.

**Review by maliha-srbd — COMMENTED** [review]
Hello @hsgwon , I have pointed out a few changes after another review. Please check them at your convenience. Thank you.

File: docs/application/dotnet/guides/multimedia/webrtc.md (hunk 2 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/webrtc.md:187]: OK I'll add `class` after `WebRTCDataChannel`.

File: docs/application/dotnet/guides/multimedia/webrtc.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/webrtc.md:497]: OK I will change it.

**Review by maliha-srbd — COMMENTED** [review]
Hi @hsgwon , hope you're well. Thank you for incorporating the suggested changes.
After another review, I have added a few more changes. Please incorporate them at your convenience. Thank you.

**Comment by maliha-srbd** [comment]
Hello @hsgwon , I hope you're doing well. This is to remind you about the [changes](https://github.com/Samsung/tizen-docs/pull/1750#pullrequestreview-1238801458) requested on this PR, please check and incorporate them at your convenience. Thank you. :)

**Review by maliha-srbd — APPROVED** [review]
Hello @hsgwon , I hope you're well. Thank you for incorporating all the suggested changes.

@asm-foysal @mijin85cho , the PR looks good to me. I'm approving it for merging.

**Pull request merged**